src/main/java/fi/otavanopisto/kuntaapi/server/discover/EntityUpdater.java

```diff
@@ -1,12 +1,34 @@
 package fi.otavanopisto.kuntaapi.server.discover;
 
-@SuppressWarnings("squid:S1610")
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.inject.Inject;
+
+import org.apache.commons.codec.digest.DigestUtils;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
 public abstract class EntityUpdater {
   
+  @Inject
+  private Logger logger;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the "logger" field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1450)